go.mod

```diff
@@ -55,6 +55,7 @@ require (
 require (
 	cloud.google.com/go v0.99.0 // indirect
 	filippo.io/age v1.0.0-beta7 // indirect
+	github.com/1Password/connect-sdk-go v1.2.0 // indirect
```

Comment:
This is a direct dependency isn't it? Wondering why it's marked as `// indirect` then